aspects/request_timeout_handler.go

```diff
@@ -0,0 +1,41 @@
+package aspects
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/config"
+	"net/http"
+	"strconv"
+)
+
+func QueuedRequestTimeout(f httprouter.Handle, custHeaders config.CustomHeaders) httprouter.Handle {
+
+	return func(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+
+		reqTimeInQueue := r.Header.Get(custHeaders.RequestTimeInQueue)
```

Comment:
I verified `Header.Get` returns an empty string and behaves nicely if one of the headers is an empty string. Looks good.